archive.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+use Collection\GenresModel;
+use Collection\RecordsModel;
+
+require_once 'vendor/autoload.php';
+require_once 'src/displayAllGenresFunction.php';
+require_once 'src/dispayAllArchivedRecordsFunction.php';
+
+//connect and format db
+$db = new PDO('mysql:host=db; dbname=collection', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+//create models
+$recordModel = new RecordsModel($db);
+$genresModel = new GenresModel($db);
+
+// get all genres
+$genres = $genresModel->getAllGenres();
+
+// Handle user input
+$CurrentrecordId  = $_POST['recordIDUpdate'] ?? false;
+$genreFilterID = $_POST['selectGenre'] ?? null;
+
+//handle return record request
+if (isset($_POST['return'])) {
+    $selectedRecordID = $_POST['recordID'];
+    if ($recordModel->returnRecord($selectedRecordID)) {
+        $recordModel->returnRecord($selectedRecordID);
```

Comment:
This is going to return the record twice, you can just remove it from line 29 and keep it in the conditional